cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaBrokerConfigurationDiff.java

```diff
@@ -171,18 +183,19 @@ private Collection<AlterConfigOp> diff(int brokerId, String desired,
                     .findFirst();
 
             String op = d.get("op").asText();
+            boolean isCombined = nodeRef.controller() && nodeRef.broker();
```

Comment:
I would be explicit about "what combined means here" by renaming the variable as `isCombinedMode`.
While it's clear on this line, it could be not clear anymore when using the variable with the various methods (i.e. `removeProperty`, ...).
Of course, I would rename the parameter for the methods using this as well.